crates/solver-service/src/apis/quote/cost/engine.rs

```diff
@@ -85,6 +72,7 @@ impl CostEngine {
 		solver: &SolverEngine,
 		config: &Config,
 		pricing: &PricingConfig,
+		pricing_service: &PricingService,
```

Comment:
We should remove pricingConfig here, since it should have been passed to PricingService already.

So anything requiring pricingConfig should be accessible internally by the service.